api/src/main/java/io/strimzi/api/kafka/model/common/template/PodTemplate.java

```diff
@@ -53,6 +54,8 @@ public class PodTemplate implements HasMetadataTemplate, UnknownPropertyPreservi
     private String priorityClassName;
     private String schedulerName;
     private List<HostAlias> hostAliases;
+    private PodDNSConfig dnsConfig;
+    private String dnsPolicy;
```

Comment:
I think this should have an Enum for the supported values. Check for example how https://github.com/strimzi/strimzi-kafka-operator/blob/0c334d8100a8c0273f28e4914a4f09fc0fa0fbb1/api/src/main/java/io/strimzi/api/kafka/model/common/template/ExternalTrafficPolicy.java#L12 is used.

Reply:
Addressed in [9bd12a2](https://github.com/strimzi/strimzi-kafka-operator/pull/10949/commits/9bd12a2eef18ffc7a3276b76ef101f326df94045)